src/schemathesis/lazy.py

```diff
@@ -66,6 +75,10 @@ def get_schema(
         method = schema.method
     if endpoint is NOT_SET:
         endpoint = schema.endpoint
+    if isinstance(schema, SwaggerV20):
```

Comment:
Now I am thinking that probably it will be easier to put `tag` field to the `BaseSchema`. To simplify everything - we can address the difference when we will actually implement support of other schema types